File: share/RunScript/PlaceFamilyAtCoordinate.cs
using Autodesk.Revit.DB;
using SCaddins.PlaceCoordinate;
using System.Collections.Generic;

public static void Main(Document doc)
{      
        var families = Command.GetAllFamilySymbols(doc);
        var family = Command.TryGetDefaultSpotCoordFamily(families);
        var location = new XYZ(0,0,0);
        Command.PlaceFamilyAtCoordinate(doc, family, location, true);
}

File: share/RunScript/PlaceFamilyAtCoordinate.lua
import ('RevitAPI', 'Autodesk.Revit.DB')
import ('System', 'System.Collections.Generic')
import ('SCaddins', 'SCaddins.PlaceCoordinate')

doc = commandData.Application.ActiveUIDocument.Document

families = Command.GetAllFamilySymbols(doc)
family = Command.TryGetDefaultSpotCoordFamily(families)
location = XYZ(0,0,0)
Command.PlaceFamilyAtCoordinate(doc, family, location, false)


